runtime/src/storable_accounts.rs

```diff
@@ -100,18 +100,22 @@ impl<'a, T: ReadableAccount + Sync> StorableAccounts<'a, T>
 }
 
 /// this tuple contains slot info PER account
-impl<'a, T: ReadableAccount + Sync> StorableAccounts<'a, T>
-    for (Slot, &'a [(&'a Pubkey, &'a T, Slot)], IncludeSlotInHash)
+impl<'a> StorableAccounts<'a, StoredAccountMeta<'a>>
```

Comment:
note: this is less generic. Seems fine - looks like we only did this for StoredAccountMeta